simcon_project/templates/update_node_response.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit Feedback</title>
+</head>
+<body>
+    {% block content %}
+        <h1>Give feedback for: {{feedback_instance.parent_template_response.student}}</h1>
+        <p>Response Transcription: {{ feedback_instance.transcription }}</p>
+        <audio src="../../../media/{{feedback_instance.audio_response}}" controls></audio>
+
+        <form action="" method="post">
+            {% csrf_token %}
+            <table>
+            {{ form.as_table }}
+            </table>
+            <input type="submit" value="Submit">
+        </form>
+    {% endblock %}
+</body>
+</html>
```

Comment:
Nit: Add a new line to the end of the file so we get rid of the warning/ good practice